index.js

```diff
@@ -0,0 +1,110 @@
+const containerElement = document.getElementById("container");
+const registerBtnElement = document.getElementById("register");
+const loginBtnElement = document.getElementById("login");
+const formLoginBtnElement = document.getElementById("loginBtn");
+const formRegistrationBtnElement = document.getElementById("registerBtn");
+
+registerBtnElement.addEventListener("click", () => {
+  container.classList.add("active");
+});
+
+loginBtnElement.addEventListener("click", () => {
+  container.classList.remove("active");
+});
+
+formLoginBtnElement.addEventListener("click", (event) => {
+  event.preventDefault();
+
+  const emailElement = document.getElementById("loginEmail").value;
+  const passwordElement = document.getElementById("loginPassword").value;
+
+  fetch("http://localhost:3000/clients")
+    .then((result) => {
+      if (!result.ok) {
+        alert("Something went wrong");
+        console.log(result);
```

Comment:
Why this console.log is needed. If it was added for debug purpose it should be removed before pushing the code